### PR TITLE
Monthly newsletter for May

### PR DESCRIPTION
**Pull request by awvwgk**
It is again time for a monthly newsletter. I put together a draft with the repository activities for May.

- [x] add stdlib updates
- [x] add fpm updates
- [x] add fortran-lang.org updates
- [x] add monthly call link
- [x] add LLVM / classic flang updates (@AlexisPerry)
- [x] add LFortran updates (@certik)
- [x] update contributor data (@LKedward)

As usual, please add your name to the authors list if you add, edit, or review content.

### CODE REVIEW

**Comment by milancurcic** [comment]
Thank you, @awvwgk!

**Comment by milancurcic** [comment]
@certik Can we have LFortran updates in tomorrow or can this newsletter go without? Thank you!

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+* We had our 11th Fortran Monthly call on April 22.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/D107yFcuZoE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+* Registration is open for the upcoming free webinar on
+  [Fortran for High Performance Computing](https://register.gotowebinar.com/register/7343048137688004108).
+  The webinar is organized by [Excellerat](https://www.excellerat.eu/)
+  and will be presented by Wadud Miah ([@wadudmiah](https://github.com/wadudmiah)) from the University of Southampton.
+
```
Comment (milancurcic) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:217]: This has passed so we should remove the note.

```suggestion
```

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+* We had our 11th Fortran Monthly call on April 22.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/D107yFcuZoE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+* Registration is open for the upcoming free webinar on
+  [Fortran for High Performance Computing](https://register.gotowebinar.com/register/7343048137688004108).
+  The webinar is organized by [Excellerat](https://www.excellerat.eu/)
+  and will be presented by Wadud Miah ([@wadudmiah](https://github.com/wadudmiah)) from the University of Southampton.
```
Comment (milancurcic) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:216]: Edit with the link to slides and recording, the event has passed.

```suggestion
Wadud Miah ([@wadudmiah](https://github.com/wadudmiah)) from the University of Southampton presented a webinar on Fortran for High Performance Computing, organized by [Excellerat](https://www.excellerat.eu/). You can find the slides and the recording [here](https://services.excellerat.eu/viewevent/39).
```

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+
+## Events
+
+* We had our 11th Fortran Monthly call on April 22.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/D107yFcuZoE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Wadud Miah ([@wadudmiah](https://github.com/wadudmiah)) from the University of Southampton presented a webinar on Fortran for High Performance Computing, organized by [Excellerat](https://www.excellerat.eu/). You can find the slides and the recording [here](https://services.excellerat.eu/viewevent/39).
```
Comment (milancurcic) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:213]:
```suggestion
* Wadud Miah ([@wadudmiah](https://github.com/wadudmiah)) from the University of Southampton presented a webinar on Fortran for High Performance Computing, organized by [Excellerat](https://www.excellerat.eu/). You can find the slides and the recording [here](https://services.excellerat.eu/viewevent/39).
```

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title: "Fortran newsletter: May 2021"
+category: newsletter
+author: Sebastian Ehlert, Alexis Perry-Holby
```
Comment (milancurcic) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:5]:
```suggestion
author: Sebastian Ehlert, Alexis Perry-Holby, Milan Curcic
```

**Review by milancurcic — APPROVED** [review]

**Comment by certik** [comment]
I'll submit the LFortran updates this afternoon.

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+* Handle instantiation of procedure pointer components
+* Fix checking of argument passing for parameterized derived types
+* Fix spurious errors from runtime derived type table construction
+* Check for attributes specific to dummy arguments
+* Handle structure constructors with forward references to PDTs
+
+Call notes will be sent to the _flang-dev_ email list and also recorded [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY).
+
+### LFortran
+
```
Comment (certik) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:204]: * 98 Merge Requests merged in April
* Working towards compiling the [SNAP](https://github.com/lanl/SNAP) proxy app ([#313](https://gitlab.com/lfortran/lfortran/-/issues/313)):
  * Code can be parsed to AST and transformed back to source code which compiles with other compilers and works
  * About 3rd of the files can be transformed from AST to ASR and the modules saved
* Other improvements:
  * Runtime library (more functions work)
  * Nested functions
  * Derived types

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "Fortran newsletter: May 2021"
+category: newsletter
+author: Sebastian Ehlert, Alexis Perry-Holby, Milan Curcic
```
Comment (certik) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:5]:
```suggestion
author: Sebastian Ehlert, Alexis Perry-Holby, Milan Curcic,  Ondřej Čertík
```

File: _posts/2021-05-01-Fortran-Newsletter-May-2021.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+* Define missing & needed IEEE_ARITHMETIC symbols
+* Handle instantiation of procedure pointer components
+* Fix checking of argument passing for parameterized derived types
+* Fix spurious errors from runtime derived type table construction
+* Check for attributes specific to dummy arguments
+* Handle structure constructors with forward references to PDTs
+
+Call notes will be sent to the _flang-dev_ email list and also recorded [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY).
+
+### LFortran
```
Comment (certik) [inline _posts/2021-05-01-Fortran-Newsletter-May-2021.md:203]:
```suggestion
### LFortran

```

**Review by certik — APPROVED** [review]
Looks good to me.

**Comment by awvwgk** [comment]
Thanks a lot everybody. I'm going ahead and merge this.

**Pull request merged**